src/Kris/LaravelFormBuilder/Fields/FormField.php

```diff
@@ -223,7 +227,7 @@ protected function prepareOptions(array $options = [])
             $this->addErrorClass();
         }
 
-        if ($this->getOption('required') === true) {
+        if ($this->getOption('required') === true || $rulesParser->containsRequired($rules)) {
```

Comment:
There shouldn't be any problem with this in the future. So please change it as i explained.
Just use `isset($rules['required'])` instead of `in_array()` or `array_search()`, key is much more important.